src/ui/components/Widgets/Chart/Chart.tsx

```diff
@@ -281,6 +282,7 @@ export const Chart = (props: ChartNoWidgetProps) => {
                 isWidgetMenuDataChanged={isWidgetMenuDataChanged}
                 renderPluginLoader={renderPluginLoader}
                 paneSplitOrientation={paneSplitOrientation}
+                dashkitConfig={dashkitConfig}
```

Comment:
Now the setting initiates the rerender of all charts - regardless of the need. Not very good